docs/pages/eas-update/code-signing.mdx

```diff
@@ -26,7 +26,11 @@ The `expo-updates` library supports end-to-end code signing. Code signing allows
      --certificate-common-name "My App"
    ```
 
-   The generated private key must be kept private and secure.
+   - The generated private key must be kept private and secure.
+   - The certificate validity duration is a setting that may vary based on the security needs of your app.
+      - A shorter validity duration will require [key rotation](#key-rotation) more frequently, but is considered better practice since a compromised private key will have a sooner expiration which limits exposure.
+      - Shorter validity durations add overhead to your app's release process as the key must be rotated more frequently. Binaries with expired certificates won't apply new updates.
+      - As an example, Expo sets this value to 20 years for the public Expo Go app, but only 1 year for internal apps with binaries that are distributed more frequently. We plan to rotate our keys every 10 years.
```

Comment:
```suggestion
   - The generated private key must be kept private and secure.
   - The certificate validity duration is a setting that may vary based on the security needs of your app.
      - A shorter validity duration will require [key rotation](#key-rotation) more frequently but is considered better practice since a compromised private key will have a sooner expiration which limits exposure.
      - Shorter validity durations add overhead to your app's release process as the key must be rotated more frequently. Binaries with expired certificates won't apply new updates.
      - For example, Expo sets this value to 20 years for the public Expo Go app, but only 1 year for internal apps with binaries that are distributed more frequently. We plan to rotate our keys every 10 years.
```